ADOPTERS.md

```diff
@@ -0,0 +1,10 @@
+# Adopters
+
+The following enumerates organizations who have adopted the AIPs in some form (directly, fork, or otherwise).
+
+If you are using the AIPs at your organization, please add your organization's name to this list (in alphabetical order).
+
+| Organization                       | Contact                                          | How you use the AIPs                                                                                                                                                                           |
+| ---------------------------------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Google](https://about.google/)    | [@toumorokoshi](https://github.com/toumorokoshi) | Google maintains a fork at [google.aip.dev](http://google.aip.dev) which is adhered to for all APIs. Google also runs the [api-linter](https://linter.aip.dev/) across the company internally. |
+| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AIPs, and is using them as a style guide for new APIs. Roblox also maintains a private fork of the [api-linter](https://linter.aip.dev).                |
```

Comment:
```suggestion
| [Roblox](https://corp.roblox.com/) | [@rofrankel](https://github.com/rofrankel)       | Roblox maintains a private fork of the AIPs, and is using them as a style guide for new APIs. Roblox also maintains private forks of the [api-linter](https://linter.aip.dev) and [site generator](https://github.com/aip-dev/site-generator).                |
```

I don't know if this is an interesting detail to include, but it might be helpful if someone wants to replicate what we've done.

Reply:
I think it's valuable to explain how it's being used, I imagine it's partially to get a sense of a the use cases.